src/index.ts

```diff
@@ -1,7 +1,6 @@
-import { defineInterface } from '@directus/utils';
 import InterfaceGroupTabs from './group-tabs.vue';
 
-export default defineInterface({
+export default {
```

Comment:
Whats the reason for removing this? As far as I can see `defineInterface` is available in the `@directus/extension-sdk` and I think it should be kept.

Reply:
This is actually the main reason for what causes such a large bundle size. I have raised the issue with Directus and they have been looking into it. Will probably be fixed in an upcoming release. Until then without it still works.

Reply:
Good to know! Thanks for raising that issue.

Reply:
Hmmmm, after trying both versions (with and without defineInterface) locally I don't see any difference in bundle size for me. But I'll accept it like this and look into it after the fact.